recipes/tensorboard_plugin_wit/meta.yaml

```
@@ -0,0 +1,36 @@
+{% set version = "1.6.0" %}
+
+package:
+  name: tensorboard-plugin-wit
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/py3/t/tensorboard-plugin-wit/tensorboard_plugin_wit-{{ version }}.post3-py3-none-any.whl
+  sha256: 1fdf4ac343f1665453205aef8bb227b0204893bb5ffb792d2ed4509b1daf3d4f
+
+build:
+  number: 0
+  noarch: python
+  script:
+    - python -m pip install --no-deps --ignore-installed tensorboard_plugin_wit-{{ version }}.post3-py3-none-any.whl
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - tensorboard_plugin_wit
+
+about:
+  home: https://pair-code.github.io/what-if-tool
+  license: Apache-2.0
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

No need, this will be found automatically if it's in the tarball
```suggestion
  license_file: LICENSE
```

---

ok